### DISCUSSION TITLE
Search - Not finding results for "Internal Reference Number"

### DISCUSSION THREAD

**Discussion by hiltonmw** [firefly-iii/firefly-iii#7789] [General]
I'm using "Internal Reference Number" as a cross-reference.  Using the search at the top left of the page yields no results when searching by known-good reference numbers.

Example : 112-6033855-5377001

That particular example has a title/description of "Rechargeable Batteries" - and I can search and find the reference number that way, however, I need to find them by Internal Reference number as well.

Is there any way to search by Internal Reference Number?  Or, could it be added as a feature request?

**Comment by JC5** [comment]
https://docs.firefly-iii.org/firefly-iii/features/search/

**Comment by hiltonmw** [comment]
Any chance we could make this easier, by allowing it to search that by default, so I don't need to use the pre-pending tags?
Or, perhaps, adding and "advanced search" page, where we could use selection boxes / check boxes, to search these fields...
Would be a nice added feature.  I'd love to help code it, but I'm rubbish at coding.

**Reply by JC5** [reply]
Way ahead of you, is already on the list. 👍🏻

**Reply by hiltonmw** [reply]
Thank you!
